magma/clock.py

```diff
@@ -95,6 +95,13 @@ def driven_implicitly_by_when(self):
     def wire(self, o, debug_info=None, driven_implicitly_by_when=False):
         if get_debug_mode() and debug_info is None:
             debug_info = get_callee_frame_info()
+        if self._driven_implicitly_by_when and not driven_implicitly_by_when:
+            # In this case, the enable value was previously implicitly driven by
+            # a when statement, but now the user is explicitly wiring the enable
+            # When this happens, we choose the user's explicit wiring, and
+            # discard the implicit logic (rather than forcing the user to call
+            # rewire explicitly)
```

Comment:
nit
```suggestion
            # In this case, the enable value was previously implicitly driven by
            # a when statement, but now the user is explicitly wiring the enable
            # When this happens, we choose the user's explicit wiring, and
            # discard the implicit logic (rather than forcing the user to call
            # rewire explicitly).
```